lib/views/frontend/spree/checkout/payment/_applepay.html.erb

```diff
@@ -0,0 +1,89 @@
+<style>
+  #apple-pay-button {
+    display: none;
+    background-color: black;
+    background-image: -webkit-named-image(apple-pay-logo-white);
+    background-size: 100% 100%;
+    background-origin: content-box;
+    background-repeat: no-repeat;
+    width: 100%;
+    max-width: 150px;
+    height: 20px;
+    padding: 8px 0;
+    border-radius: 4px;
+  }
+  @media screen and (max-width: 767px) {
+    #apple-pay-button {
+      margin: 0 auto;
+    }
+  }
+</style>
+<span id="apple-pay-button"></span>
+
+<script type="text/javascript" src="https://js.stripe.com/v2/"></script>
```

Comment:
@harsh-ag If stripe gateway is enable then `https://js.stripe.com/v2/` will be loaded twice on same page. Also stripe publishable key will be assigned twice.
We should check if these are loaded/assigned respectively and take action accordingly.